ls1intum/Artemis
# [Feature] Adding a Swift template programming exercise

### Checklist

### Motivation and Context


We want to have Swift exercises in Artemis, so that it also might be used in the iPraktikum.

### Description

This PR adds a template for Swift programming exercises but doesn't contain valid tests for the CI, yet.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a new Programming Exercise and select Swift in the dropdown menu
4. Verify it was successful and start the exercise
4. Click on open in Online Editor 
5. See a working template for BubbleSort and MergeSort as a Swift code exercise

The testsuite will follow in a later PR. 

### Test Coverage


### Screenshots



Note: The new programming languages are only supported with Bamboo, so make sure to use TS1 or TS3 when testing.
